peps/pep-9999.rst

```diff
@@ -0,0 +1,557 @@
+PEP: 9999
+Title: Calendar versioning for Python
+Author: Hugo van Kemenade
+Discussions-To: <REQUIRED: URL of current canonical discussion thread>
+Status: Draft
+Type: Process
+Created: 20-May-2024
+Python-Version: 3.26
+Post-History: <REQUIRED: dates, in dd-mmm-yyyy format, and corresponding links to PEP discussion threads>
+
+
+Abstract
+========
+
+This PEP proposes updating the versioning scheme for Python to include
+the calendar year.
+This aims to make the support lifecycle clear
+by making it easy to see when a version was first released,
+and easier to work out when it will reach end-of-life.
```

Comment:
"end of life" should be hyphenated when it's used as an adjective, but here it's being used as a noun, so the hyphens are unnecessary :)

Also: you use the EOL initialism later in the PEP; let's define that here.

```suggestion
and easier to work out when it will reach end of life (EOL).
```